app/dashboard/static/js/app/view-tests-all.2018.9.js

```diff
@@ -283,45 +163,56 @@ require([
         } else {
             columns = [
                 {
-                    data: 'board',
-                    title: 'Board',
+                    data: 'job',
+                    title: 'Tree',
                     type: 'string',
-                    render: ttest.renderBoard
+                    className: 'tree-column',
+                    render: _renderTree
                 },
                 {
-                    data: 'board',
-                    title: 'Total Test Groups',
-                    type: 'num',
-                    searchable: false,
-                    className: 'pull-center',
-                    render: _renderGroupsCount
+                    data: 'git_branch',
+                    title: 'Branch',
+                    type: 'string',
+                    className: 'branch-column',
+                    render: _renderBranch
                 },
                 {
-                    data: 'board',
-                    title: 'Total Unique Labs',
-                    type: 'num',
-                    searchable: false,
-                    className: 'pull-center',
-                    render: _renderLabsCount
+                    data: 'kernel',
+                    title: 'Kernel',
+                    type: 'string',
+                    className: 'kernel-column',
+                    render: _renderKernel
                 },
                 {
-                    data: 'board',
-                    title: '',
+                    data: 'name',
+                    title: 'Test Plan',
+                    type: 'string',
+                    className: 'plan-column',
+                    render: _renderPlan
+                },
+                {
+                    data: 'created_on',
+                    title: 'Date',
+                    type: 'date',
+                    className: 'date-column',
+                    render: ttest.renderDate
+                },
+                {
+                    data: '_id.$oid',
+                    title: 'Status',
                     type: 'string',
                     searchable: false,
                     orderable: false,
-                    className: 'select-column pull-center',
-                    render: _renderDetails
+                    className: 'plan-center pull-center',
+                    render: _renderStatus
                 }
             ];
 
             gTestsTable
                 .data(response)
                 .columns(columns)
-                .order([0, 'asc'])
+                .order([4, 'desc'])
                 .languageLengthMenu('Tests per page')
-                .rowURL('/test/board/%(board)s/')
```

Comment:
Actually, maybe this URL could be the same as in the "plan" column to go to a URL like `https://staging.kernelci.org/test/job/kernelci/branch/staging.kernelci.org/kernel/staging-20200408.1/plan/baseline`
so `/test/job/%(job)s/kernel/%(kernel)s/plan/$(plan)s/`

Reply:
sure, do you mean : /test/job/%(job)s/branch/%(branch)s/kernel/%(kernel)s/plan/%(plan)s/ or a new view? I updated to go to the link that we already have.

Reply:
Yes, with `/branch/` in the middle... sorry these URLs are rather long.